plugin/trino-kinesis/src/main/java/io/trino/plugin/kinesis/KinesisConfig.java

```diff
@@ -28,6 +29,7 @@
     private String defaultSchema = "default";
     private String tableDescriptionLocation = "etc/kinesis/";
     private boolean hideInternalColumns = true;
+    private Duration updateInterval = new Duration(1, TimeUnit.MINUTES);
```

Comment:
Let's rename to `tableDescriptionRefreshInterval` and the relevant places. Also, move to directly under `tableDescriptionLocation`.